backend/private-graph/graph/resolver.go

```diff
@@ -1989,6 +1994,24 @@ func (r *Resolver) RemoveJiraFromWorkspace(workspace *model.Workspace) error {
 	return nil
 }
 
+func (r *Resolver) RemoveGitlabFromWorkspace(workspace *model.Workspace) error {
+	ctx := context.TODO()
+	accessToken, err := r.IntegrationsClient.GetWorkspaceAccessToken(ctx, workspace, modelInputs.IntegrationTypeGitlab)
```

Comment:
I try to "revoke" existing token. This is a "best effort" attempt. 
The integration is removed from the workspace regardless of whether this succeeds.